src/main/kotlin/mate/academy/HelloWorldAnonymousClasses.kt

```diff
@@ -8,6 +8,45 @@ class HelloWorldAnonymousClasses {
     }
 
     fun sayHello(names: List<String>): List<String> {
-        // TODO: implement
+        val englishGreeting = object : HelloWorldGreeting {
+            override fun greet(): String {
+                return "Hello world"
+            }
+            override fun greetSomeone(someone: String): String {
+                return "Hello $someone"
+            }
+        }
+
+        val frenchGreeting  = object : HelloWorldGreeting {
+            override fun greet(): String {
+                return "Salut tout le monde"
```

Comment:
It is better to move string values to constants of sting res (in case of Android app development)